setup.py

```diff
@@ -119,7 +119,10 @@ def get_blas_lib_dir(cls):
     'torch>=2.0; python_version>="3.8"',
     'sentencepiece>=0.1.86,!=0.1.92', # 0.1.92 results in error for transformers
     'transformers>=4.1.1; python_version<"3.9"',
-    'transformers>=4.4.2; python_version>="3.9"'
+    'transformers>=4.4.2; python_version>="3.9"',
+    'tqdm>=4.66.4',
+    'peft>=0.11.0',
```

Comment:
This package requires `python_version >= 3.8` that's why the latest unit test failed.